tests/stdlib/tmersenne.nim

```diff
@@ -0,0 +1,11 @@
+import std/mersenne
+
+template main() =
```

Comment:
in future work, it would be nice to add a test that checks that we generate a uniform distribution (many ways to do it eg by counting generated numbers modulo a small K and checking that each count is >= 1/K - eps for large enough sample size; but preferably using something standard); the same code should be reused for trandom and upcoming tsecrets (https://github.com/nim-lang/Nim/pull/16459)